### PR TITLE
Move the accesibility placeholder to the platform dispatcher.

### PR DESCRIPTION
**Pull request by tugorez**
Move the accesibility placeholder to the platform dispatcher.

This change makes the platform dispatcher append a single accesibility placeholder, per app, to the `<body />`. Previous behavior was to insert a placeholder inside each `<flutter-view />`

Relevant Issues are:

* Design doc: https://flutter.dev/go/focus-management 
* Focus in web multiview: https://github.com/flutter/flutter/issues/137443
* Platform dispatcher changes: https://github.com/flutter/engine/pull/49841

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or the PR is [test-exempt]. See [testing the engine] for instructions on writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/main/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: lib/web_ui/lib/src/engine/platform_dispatcher.dart [hunk 1]
```diff
@@ -92,6 +93,10 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
   /// The [EnginePlatformDispatcher] singleton.
   static EnginePlatformDispatcher get instance => _instance;
   static final EnginePlatformDispatcher _instance = EnginePlatformDispatcher();
+  static final DomElement _accessibilityPlaceholder = EngineSemantics
```
Comment (yjbanov) [inline lib/web_ui/lib/src/engine/platform_dispatcher.dart:96]: Can we remove `static`? The `PlatformDispatcher` is already a singleton that should guarantee there's no more than one placeholder. However, being non-static would guarantee that the lifecycle of the placeholder is attached to the lifecycle to `PlatformDispatcher`, such as when hot-reload/hot-restart happens, or when/if we support shutting down and reinitializing the engine.

File: lib/web_ui/lib/src/engine/platform_dispatcher.dart [hunk 2]
```diff
@@ -109,13 +114,15 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     return builder.build();
   }
 
+
```
Comment (yjbanov) [inline lib/web_ui/lib/src/engine/platform_dispatcher.dart:117]: nit: do we need this extra blank line?

Comment (tugorez) [reply lib/web_ui/lib/src/engine/platform_dispatcher.dart:117]: Great catch. Removed!

**Comment by tugorez** [comment]
https://flutter-web-20c94.web.app this is a testing app with these changes that can be used to perform manual verifications 

cc @yjbanov

**Review by yjbanov — APPROVED** [review]
lgtm

**Pull request merged**